File: 백준/9단계/소인수분해.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n')

let N = Number(input[0])
// len N = 72

let i = 2;
while (N != 1) {
    if (N % i === 0) {
        console.log(i)
        N = N / i
    } else {
        i += 1
    }
}

File: 백준/9단계/소인수분해.py
N = int(input())

# N의 소인수분해 결과를 한줄에 하나씩 오름차순으로 출력
i = 2
while N != 1:
    if N % i == 0:
        N = N // i
        print(i)
    else:
        i += 1
